lib/nostrum/api.ex

```diff
@@ -734,15 +737,32 @@ defmodule Nostrum.Api do
   def bulk_delete_messages(channel_id, messages, filter \\ true)
 
   def bulk_delete_messages(channel_id, messages, false),
-    do: request(:post, Constants.channel_bulk_delete(channel_id), %{messages: messages})
+    do: send_chunked_delete(channel_id, messages)
 
   def bulk_delete_messages(channel_id, messages, true) do
     filtered_messages =
       Enum.filter(messages, fn message_id ->
         (message_id >>> 22) + 1_420_070_400_000 > Util.now() - 14 * 24 * 60 * 60 * 1000
       end)
 
-    request(:post, Constants.channel_bulk_delete(channel_id), %{messages: filtered_messages})
+    send_chunked_delete(channel_id, filtered_messages)
+  end
+
+  @spec send_chunked_delete(
+          Nostrum.Struct.Snowflake.t(),
+          [Nostrum.Struct.Message.id()]
+        ) :: error | {:ok}
+  def send_chunked_delete(channel_id, messages) do
+    messages
+    |> Enum.chunk_every(100)
```

Comment:
It does matter if this is a stream function, I think it should be.

Reply:
Changed to `Stream.chunk_every(100)` in f3d8061f4d75fa892046d273f19ed29624251c09.